### PR TITLE
Enable gas price

### PR DESCRIPTION
**Pull request by SatpalSandhu61**
Allow GoQuorum to be set up as a gas enabled network.

Associated changes and pull requests:

- [Documentation](https://github.com/ConsenSys/doc.goquorum/pull/244).
- Acceptance tests (to follow).

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -8,6 +8,7 @@ on:
       - '.gitignore'
     branches:
       - master
+      - enable-gas-price-v2
```
Comment (baptiste-b-pegasys) [inline .github/workflows/build.yml]: to remove before merge in master

File: consensus/ethash/consensus.go [hunk 2]
```diff
@@ -624,14 +624,18 @@ var (
 // reward. The total reward consists of the static block reward and rewards for
 // included uncles. The coinbase of each uncle block is also rewarded.
 func accumulateRewards(config *params.ChainConfig, state *state.StateDB, header *types.Header, uncles []*types.Header) {
+
+	//NOTE: this code will be executed for Raft on Quorum, so the static block reward also applies to Raft
+	blockReward := big.NewInt(0)
 	// Select the correct block reward based on chain progression
-	blockReward := FrontierBlockReward
+	blockReward = FrontierBlockReward
```
Comment (baptiste-b-pegasys) [inline consensus/ethash/consensus.go]: a condition to add here?

Comment (SatpalSandhu61) [reply consensus/ethash/consensus.go]: I had added a condition, but didn't need it in the end. It looks like I left the partial change there - will correct it.

File: .github/workflows/build.yml (hunk 1 above)
Comment (SatpalSandhu61) [reply .github/workflows/build.yml]: Done

File: consensus/ethash/consensus.go (hunk 2 above)
Comment (SatpalSandhu61) [reply consensus/ethash/consensus.go]: This has been corrected now

**Review by baptiste-b-pegasys — APPROVED** [review]

**Review by antonydenyer — APPROVED** [review]

**Pull request merged**